pkg/controller/nodenetworkstate/nodenetworkstate_controller.go

```diff
@@ -36,8 +41,22 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	forThisNode := predicate.Funcs{
+		CreateFunc: func(createEvent event.CreateEvent) bool {
+			return nmstate.EventIsForThisNode(createEvent.Meta)
+		},
+		DeleteFunc: func(deleteEvent event.DeleteEvent) bool {
+			return nmstate.EventIsForThisNode(deleteEvent.Meta)
+		},
+		UpdateFunc: func(updateEvent event.UpdateEvent) bool {
+			return nmstate.EventIsForThisNode(updateEvent.MetaNew)
+		},
+		GenericFunc: func(genericEvent event.GenericEvent) bool {
```

Comment:
what is this GenericFunc? Do we care about it?

Reply:
It's for events that are unknown, maybe refresh is one of them.